scripts/start-on-boot.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -ex
+here=$(dirname $(dirname $(realpath $0)))
+target=/etc/systemd/system/montagu-ci.service
+read -p "What user should the service run as? " user
```

Comment:
service -> "virtual machines" I think (I had to think about this when trying it out)

Reply:
It should probably also default to the current user if that's not too much of a faff to do in bash

Reply:
I originally had it default to the current user, but then that's always root, as you need root to set up the service. For a while I had it get the current user and then `sudo` inside the script, but that felt like an anti-pattern. Can you think of a better way?

Fixed the question here https://github.com/vimc/montagu-ci/pull/13/commits/2ecc44372c34a1cf04dac996a20a3de8b43974df